sbt-ci-openjdk8/Dockerfile

```diff
@@ -1,31 +1,32 @@
-FROM adoptopenjdk/openjdk8:x86_64-ubuntu-jdk8u252-b09-slim
+FROM adoptopenjdk/openjdk8:jdk8u292-b10-ubuntu-slim
 
-ARG SBT_VERSION=1.3.10
+ARG SBT_VERSION=1.5.5
 
-LABEL version="jdk8u252-b09-$SBT_VERSION"
+LABEL version="jdk8u292-b10-$SBT_VERSION"
 LABEL maintainer="Megumu Katsuno <katsuno@chatwork.com>"
 
 ENV LANG=C.UTF-8
 
-RUN curl -L -o /tmp/sbt.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
-    dpkg -i /tmp/sbt.deb && \
-    rm /tmp/sbt.deb && \
+RUN apt-get update -y && apt-get install -y gnupg2 && \
+    echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | tee /etc/apt/sources.list.d/sbt.list && \
+    echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee /etc/apt/sources.list.d/sbt_old.list && \
+    curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add && \
     apt-get update && \
     apt-get install -y \
     sbt \
     libaio1 \
     git \
     unzip \
-    python \
+    python3 \
```

Comment:
Due to the end of support for Python 2.7 on AWS CLI v1
https://aws.amazon.com/jp/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/